omnibus-snaxchain.toml

```diff
@@ -0,0 +1,33 @@
+name = "synthetix-omnibus"
+version = "1"
+description = "Snaxchain deployment"
+include = [
+    # Governance
+    "tomls/governance/councils-mothership.toml",
+    "tomls/governance/wormhole.toml",
+]
+
+[setting.owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9" # TODO: confirm owner address
+
+# Governance Councils Configuration
+[var.governance]
+governance_package = "synthetix-governance:3.0.5"
+governance_snapshot_record_package = "synthetix-snapshot-rewards:3.5.0"
+governance_salt = "synthetix-governance"
+# Wormhole
+governance_wormhole_core = "0xc1BA3CC4bFE724A08FbbFbF64F8db196738665f4"    # https://explorer.snaxchain.io/address/0xc1BA3CC4bFE724A08FbbFbF64F8db196738665f4
+governance_wormhole_relayer = "0x27428DD2d3DD32A4D7f7C497eAaa23130d894911" # https://explorer.snaxchain.io/address/0x27428DD2d3DD32A4D7f7C497eAaa23130d894911
+# Proxy Config
+governance_initial_council_member = "<%= settings.owner %>"
+governance_minimum_active_members = "1"
```

Comment:
wanted to confirm the `min active members` value

Reply:
I don't think it should be `1` but maybe on snaxchain at init we set it to `1` until we elect members and then we raise to expected values. I think on OP we might want to set the existing members as the initial council members and set the minimum from the beginning but not sure.